components/layouts/layout.tsx

```diff
@@ -0,0 +1,32 @@
+import { Container, Heading, Box, Link, Flex, Spacer } from '@chakra-ui/react'
+
+interface LayoutProps {
+  children: React.ReactNode
+}
+
+const Layout = ({ children }: LayoutProps) => {
+  return (
+    <>
+      <Box padding='4'>
+        <Flex>
+          <Box p={2}>
+            <Heading size='md'>Omise – HENKAKU</Heading>
+          </Box>
+          <Spacer />
+          <Box p={2}>
+            <Link href='/' p={4} color='#3c859d' _hover={{}} borderBottom='1px'>
```

Comment:
[IMO] 
It might be better to extend colorSchema, since it would be hard to hard-code the color code each time.
https://chakra-ui.com/docs/components/navigation/link#props

Reply:
yeah true, I just copied and pasted previous one